jobs/JJOB_PREP_SOCA

```diff
@@ -38,10 +38,3 @@ echo
 mkdir -p bump
 srun -n $NPE ${SOCA_EXEC}/soca_staticbinit.x ./yaml/static_SocaError_init.yml
 
-# Initialize BUMP localization (computes localization weights)
```

Comment:
This cannot be deleted, because it is used from all the cases.

Reply:
Thanks for pointing out.
Added.